ios/Rainbow.xcodeproj/project.pbxproj

```diff
@@ -7,7 +7,7 @@
 	objects = {
 
 /* Begin PBXBuildFile section */
-		00DDBE36D4F0C321C92E55F7 /* libPods-PriceWidgetExtension.a in Frameworks */ = {isa = PBXBuildFile; fileRef = 2B57E8B9D19D57A940427008 /* libPods-PriceWidgetExtension.a */; };
```

Comment:
was this intended?

Reply:
No, ive reverted it back to whats currently on develop